engine/installation/linux/linux-postinstall.md

```diff
@@ -97,6 +97,69 @@ For information about the different storage engines, see
 The default storage engine and the list of supported storage engines depend on
 your host's Linux distribution and available kernel drivers.
 
+## Enable experimental features
+
+Some features of Docker are introduced as experimental features for one or more
+releases. Previously, you needed to download and install an
+**experimental build** to use these features. In Docker 1.13 and higher,
+experimental features are included in all stable Docker builds and packages,
+but disabled by default. To enable experimental features, start the Docker
+daemon (`dockerd`) with the `--experimental` flag. You can do this temporarily
+or permanently.
+
+> **Note**: Docker for Windows and Docker for Mac ship with experimental
+> features enabled by default.
```

Comment:
@mstanleyjones @thaJeztah The Docker for Mac and Docker for Windows Beta channel releases ship with experimental features enabled by default, and you have the option to switch this off and use the last stable release of Engine. The Stable releases now also have the option to use experimental features as an option on the Docker Daemon, but experimental features are off by default on the Stable channel. 

This is all documented but I will to update those docs just a bit to underscore the fact that you have the same options on stable and on beta (https://docs.docker.com/docker-for-mac/#/experimental-mode, https://docs.docker.com/docker-for-windows/#/experimental-mode)